RIOT-OS/RIOT
# makefiles/suit: allow to overwrite SUIT_VENDOR_DOMAIN

### Contribution description

I think the intention was that SUIT_VENDOR_DOMAIN gets set to $(SUIT_VENDOR) so it can be overwritten by the build system.

However, no such code was in place yet.


### Testing procedure

Set `SUIT_VENDOR=example.com` when building e.g. `examples/suit_update` and see how the vendor ID changes.


### Issues/PRs references
